src/modules/10-common/layouts/layouts.module.scss

```
@@ -64,25 +65,40 @@
 .trialLicenseBanner + .featuresBanner + .children {
   height: calc(100vh - 95px);
 }
+*/
 
 .rhs {
-  overflow-y: auto;
-  height: 100vh;
+  .children {
+    min-height: var(--page-min-height);
+    background-color: var(--primary-bg);
+  }
 }
 
 .main {
   display: grid;
-  grid-template-columns: 88px 184px calc(100vw - 272px);
+  grid-template-columns: 88px 184px calc(100% - 272px);
 
   &[data-layout='minimal'] {
-    grid-template-columns: 88px calc(100vw - 88px);
+    grid-template-columns: 88px calc(100% - 88px);
```

Any reason to not just use `1fr`
```suggestion
    grid-template-columns: 88px 1fr;
```